src/lib.rs

```diff
@@ -40,8 +40,10 @@
 //!                         .user("hugo"),
 //!             )?
 //!             .pre_install_script("echo preinst")
-//!             .add_changelog_entry("me", "was awesome, eh?", 123123123)
-//!             .add_changelog_entry("you", "yeah, it was", 12312312)
+//!             .build_time(std::time::SystemTime::now())
+//!             .build_host(gethostname::gethostname().to_str().unwrap().to_string())
+//!             .add_changelog_entry("me", "was awesome, eh?", std::time::SystemTime::now())
```

Comment:
I don't know that we should encourage using it like this, changelog times ought to be static and you wouldn't want multiple of them at the same timestamp.

Reply:
I just converted the example I think?

Reply:
Exactly, we're doing so in an example - that's encouraging :)